src/Illuminate/Support/Collection.php

```diff
@@ -69,6 +69,17 @@ public function all()
     }
 
     /**
+     * Get all of the items in the collection.
```

Comment:
This doesn't make sense to me since the method returns true or false, it's a test to see if any item pass the callback...